tests/_core/search/pipeline.yaml

```diff
@@ -0,0 +1,37 @@
+$schema: ../../../json_schemas/test_story.schema.yaml
+
+description: |
+  Test the creation of a search pipeline with sort and split response processors.
+epilogues:
+  - path: /_search/pipeline/sorting_pipeline
+    method: DELETE
+    status: [200, 404]
+version: '>= 2.16'
```

Comment:
The CI failed because we lock the build to a specific version on dockerhub [here](https://github.com/opensearch-project/opensearch-api-specification/blob/main/.github/workflows/test-spec.yml#L29) - update it to the latest one that also has the new pipeline added.

Reply:
> The CI failed because we lock the build to a specific version on dockerhub [here](https://github.com/opensearch-project/opensearch-api-specification/blob/main/.github/workflows/test-spec.yml#L29) - update it to the latest one that also has the new pipeline added.

The PR adding this feature has not been merged yet.  What is the usual pattern for writing a test for unmerged code, since submitting this API PR is a prerequisite to merging the PR? 🐔 🥚

Reply:
I think you have two choices.

1. Wait till the PR is merged and there's a build with it.
2. Split this PR in 2, one for existing features and one for the new one, and (1) for the new one.

Reply:
I had already assumed option 1 was the expectation.

Reply:
Thanks @dbwiddis, looking forward to getting all this closed out!